pkg/virt-launcher/virtwrap/network/dhcp/dhcp.go

```diff
@@ -46,12 +46,12 @@ func SingleClientDHCPServer(
 	handler := &DHCPHandler{
 		clientIP:      clientIP,
 		clientMAC:     clientMAC,
-		serverIP:      serverIP,
+		serverIP:      serverIP.To4(),
 		leaseDuration: infiniteLease,
 		options: dhcp.Options{
 			dhcp.OptionSubnetMask:       []byte(clientMask),
 			dhcp.OptionRouter:           []byte(routerIP),
```

Comment:
And here we don't need a`routerIP.To4()`?

Reply:
> @vladikr what about routerIP and clientMask?

Ok, I see in the data you shared in #789 that the subnet mask seems to be serialized to four byte, let's check that routerIP is ok too.

Reply:
> let's check that routerIP is ok too.

Yea, Default-Gateway Option 3, length 4: 10.244.0.1
OptionRouter                 OptionCode = 3

Reply:
Great!